app/src/app/api/v0/organizations/route.ts

```diff
@@ -5,6 +5,7 @@ import { NextResponse } from "next/server";
 import UserService from "@/backend/UserService";
 import { apiHandler } from "@/util/api";
 import { OrganizationInvite } from "@/models/OrganizationInvite";
```

Comment:
I think this import can be removed now since you're not using it directly anymore?